src/cobbler_tftp/settings/schemas/v1_0.py

```diff
@@ -0,0 +1,68 @@
+"""
+Configuration for version 1.0
+"""
+
+from pathlib import Path
+from typing import Dict, Union
+
+from schema import Optional, Schema, SchemaError
+
+settings_schema: Schema = Schema(
+    {
+        Optional("schema"): float,
+        Optional("auto_migrate_settings"): bool,
+        Optional("is_daemon"): bool,
+        Optional("cobbler"): {
+            Optional("uri"): str,
+            Optional("username"): str,
+            Optional("password"): str,
+            Optional("password_file"): str | Path,
+        },
+    }
+)
+
+
+def validate_config(
```

Comment:
```suggestion
def validate(
```

To stay in line with Cobbler I would kindly ask that you remove the `config` suffix from this method.

Reply:
Well again nothing wrong with your argumentation there but if you are familiar with the application it shouldn't be a problem to distinguish them. If you are not then probably the difference doesn't matter. Do as you please in this regard. It was just a nitpick.

Reply:
Fair point. However, as this project will be open-source at some point (technically already is) I personally would prefer to make it as easy as possible for outside contributors unfamiliar to the project to get into it; however unrealistic that may be in practice. I totally see your point though: Adopting naming conventions from Cobbler would make it easier for people familiar with Cobbler to contribute to cobbler-tftp aswell. We would have to weigh how our priorities would look on that end: better for outsiders or better for established Cobbler contributors.

Thinking of it, as this is a sub-project for Cobbler your approach would indeed make more sense now. Still wanted to present the argument though.